pkgs/applications/version-management/git-and-tools/default.nix

```diff
@@ -147,39 +149,43 @@ let
     inherit (darwin) Security;
   };
 
-  hubUnstable = throw "use gitAndTools.hub instead";
+  hub-unstable = throw "use gitAndTools.hub instead";
 
   lab = callPackage ./lab { };
 
   lefthook = callPackage ./lefthook { };
 
-  pre-commit = pkgs.python3Packages.toPythonApplication pkgs.python3Packages.pre-commit;
-
   pass-git-helper = python3Packages.callPackage ./pass-git-helper { };
 
+  pre-commit = pkgs.python3Packages.toPythonApplication pkgs.python3Packages.pre-commit;
+
   qgit = qt5.callPackage ./qgit { };
 
-  stgit = callPackage ./stgit {
-  };
+  stgit = callPackage ./stgit { };
 
   subgit = callPackage ./subgit { };
 
+  svn-all-fast-export = libsForQt5.callPackage ./svn-all-fast-export { };
+
   svn2git = callPackage ./svn2git {
     git = gitSVN;
   };
 
-  svn-all-fast-export = libsForQt5.callPackage ./svn-all-fast-export { };
-
   tig = callPackage ./tig { };
 
-  topGit = callPackage ./topgit { };
+  top-git = callPackage ./topgit { };
```

Comment:
although these names should have never been merged in the first place, renaming them can be a breaking change for people who have created workflows around them. If you're going to change them, I would at least do it in another PR where we aren't conflating the addition of git-my and the issue of naming

@worldofpeace thoughts?

Reply:
It seems @bb010g established aliases so if someone relies on them the alias will make it backwards compatible.
Though it would be better in a PR, but I believe no one has issues with hyphenated attribute names anymore.

Reply:
oh, i skipped over the aliases at the bottom, LGTM now